src/main/java/com/maxmind/minfraud/request/Email.java

```diff
@@ -21,22 +22,217 @@ public final class Email extends AbstractModel {
     private final boolean hashAddress;
     private final String domain;
     private static final Map<String, String> typoDomains;
+    private static final Map<String, String> equivalentDomains;
+    private static final Map<String, Boolean> fastmailDomains;
+    private static final Map<String, Boolean> yahooDomains;
 
     static {
-        HashMap<String, String> m = new HashMap<>() {{
+        HashMap<String, String> typoDomainsMap = new HashMap<>() {{
             // gmail.com
-            put("35gmai.com", "gmail.com");
-            put("636gmail.com", "gmail.com");
+            put("gmai.com", "gmail.com");
             put("gamil.com", "gmail.com");
-            put("gmail.comu", "gmail.com");
+            put("gmali.com", "gmail.com");
             put("gmial.com", "gmail.com");
             put("gmil.com", "gmail.com");
-            put("yahoogmail.com", "gmail.com");
+            put("gmaill.com", "gmail.com");
+            put("gmailm.com", "gmail.com");
+                put("gmailo.com", "gmail.com");
```

Comment:
The indentation here is off

Reply:
Yeah, I don't know what is up with the indentation. The linter is complaining unless I make these odd indentations. I couldn't figure it out. Maybe I could annotate the lines to direct the linter to ignore it or something. I'll try that.